tests/integration/tests/api/versions.py

```
@@ -44,15 +44,18 @@ def test_list_versions_index(self):
         expected_version = test_config.values['reddwarf_version']
         assert_equal(expected_version, versions[0].id,
                      message="Version ID: %s" % versions[0].id)
+        expected_api_updated = test_config.values['reddwarf_api_updated']
+        assert_equal(expected_api_updated, versions[0].updated,
+                     message="Version updated: %s" % versions[0].updated)
 
     def _request(self, url, method='GET', response='200'):
         resp, body = None, None
+        full_url = test_config.version_url + url
         try:
-            full_url = 'http://localhost:8775' + url
             resp, body = self.client.client.request(full_url, method)
             assert_equal(resp.get('status', ''), response)
-        except Exception:
-            pass
+        except Exception as e:
+            assert_equal(str(e.http_status), response)
```

Basically, yes. Before these tests were "passing" because of except: pass, even though they should have been failing, or even exploding would have been better. Any exception that's tripped in the client that isn't a known one with http_status (which I'm expecting) won't be anything to handle here, other than failing the test and dumping it out.